<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>条形码读取</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>565</y>
         <width>111</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>关闭界面</string>
       </property>
      </widget>
      <widget class="QLabel" name="label2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>540</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>XXXXXXXXXX</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>553</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;识别结果：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_0">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="process">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>565</y>
         <width>111</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2.图片预处理</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>270</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label2_2">
       <property name="geometry">
        <rect>
         <x>105</x>
         <y>240</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>原图像显示</string>
       </property>
      </widget>
      <widget class="QLabel" name="label2_3">
       <property name="geometry">
        <rect>
         <x>385</x>
         <y>240</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sobel梯度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label2_4">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>480</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>霍夫直线</string>
       </property>
      </widget>
      <widget class="QLabel" name="label2_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>480</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>校正后显示</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>270</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>531</y>
         <width>111</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>3.开始识别</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>531</y>
         <width>111</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1.打开图片</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>505</y>
         <width>511</width>
         <height>95</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border-width:5px;   //线的粗细
   border-style:solid;
  border-color:lightGray;   //颜色，
  margin-top: 0.5ex;  //文字在方框中位置的偏离度
}</string>
       </property>
       <property name="title">
        <string>操作步骤</string>
       </property>
      </widget>
      <zorder>groupBox_2</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>label2</zorder>
      <zorder>label</zorder>
      <zorder>label_0</zorder>
      <zorder>process</zorder>
      <zorder>label_1</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>label2_2</zorder>
      <zorder>label2_3</zorder>
      <zorder>label2_4</zorder>
      <zorder>label2_5</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>pushButton</zorder>
      <zorder>start</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
